awesomerc.lua.in

```diff
@@ -270,7 +276,9 @@ globalkeys = awful.util.table.join(
                   awful.util.getdir("cache") .. "/history_eval")
               end),
     -- Menubar
-    awful.key({ modkey }, "p", function() menubar.show() end)
+    awful.key({ modkey }, "p", function() menubar.show() end),
+    -- Alt + Left Shift switches the current keyboard layout
+    awful.key({ "Mod1" }, "Shift_L", function () mykeyboardlayout.next_layout() end)
```

Comment:
I see your point. I will not change default hotkeys. Let user choose appropriate hotkey.